index.html

```diff
@@ -307,6 +307,55 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="about flex-column">
```

Comment:
- [ ] Please add styling to the form I think you forgot it so kindly style your form as mentioned by the previous reviewer. :point_up: